src/components/AdminPortal/OrganizationDashCards/DashboardCard.module.css

```
@@ -1,60 +1,97 @@
+  font-weight: var(--font-weight-medium);
+  color: var(--color-gray-mid);
+  letter-spacing: var(--letter-spacing-snug-half);
+  text-transform: capitalize;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+/* ── Unused legacy classes kept as no-ops to avoid TS errors ─────────────────── */
+.cardBodymain {
+}
```

Stylelint (17.6.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)